libnvme/src/wdc.rs

```diff
@@ -0,0 +1,33 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::{controller::LockedController, error::LibraryError, NvmeError};
+
+use libnvme_sys::nvme::{nvme_wdc_resize_get, nvme_wdc_resize_set};
+
+impl<'handle> LockedController<'handle> {
+    pub fn wdc_resize_set(&self, size: u32) -> Result<(), NvmeError> {
+        let controller =
+            self.controller.as_ref().expect("controller is locked");
```

Comment:
This shows up enough it's probably worth a private helper function (`self.inner()` or somesuch).

Reply:
Ended up reusing the `deref` call for these. Fixed in commit [1e4f35b](https://github.com/oxidecomputer/libnvme/pull/3/commits/1e4f35bc1355fec7600d62e8fb22871e96febbfa)